test/inductor/test_torchinductor_dynamic_shapes.py

```diff
@@ -575,6 +581,7 @@ def f(x):
 
         f(torch.tensor([3], device=device))
 
+    @skipCUDAIf(IS_SM89, "Fails(with OOMS) on SM89")
```

Comment:
nit: Why not expand the newly added `xFailIfSM89` to accept an optional message, so that you can use the same skip function across the board?

Reply:
Because when it OOMs, it can not `xfail`